src/js/scrollArea.jsx

```diff
@@ -148,6 +163,11 @@ export default class ScrollArea extends React.Component{
         this.setState(newState);
     }
 
+    convertLinesToPixels(lines) {
+        var pixels = lineHeight(findDOMNode(this.refs.content));
```

Comment:
Yes but if I understand, function `lineHeight()` returns constant height of line(in some DOM element). It's not depend on wheel speed so it could be compute once in componentDidMount.